mlflow/pyfunc/__init__.py

```diff
@@ -716,6 +717,24 @@ def load_model(
     return PyFuncModel(model_meta=model_meta, model_impl=model_impl)
 
 
+def print_model_dependencies(model_uri):
```

Comment:
Should this be a fluent API (e.g. `mlflow.print_model_dependencies`)?

Reply:
I don't think the fluent API is the right place for this. The `mlflow.models` namespace seems better.